translations/cs/LC_MESSAGES/wesnoth-ala.po

```diff
@@ -2678,6 +2683,10 @@ msgid ""
 "not only the defense of the town, but also its economic rebirth, committing "
 "itself to reopening the legendary Amber Road! "
 msgstr ""
+"Mladý důstojník wesnothské armády je poslán, aby velel malé posádce v "
+"bezvýznamném městečku v divočině dálného východu. Nečekaně zvládne nejen "
+"obranu města, ale i jeho ekonomické znovuzrození, zavazujíc se tak "
```

Comment:
masculine transgressive
```suggestion
"obranu města, ale i jeho ekonomické znovuzrození, zavazuje se tak "
```

Reply:
hm, let's see where the commit will go if I click the "Commit suggestion" button... to your branch, or mine?

Reply:
ok it looks like it went to Septim's branch